docs/performance.md

```diff
@@ -1,39 +1,72 @@
 
-Below are the WPS and MFU results which torchtitan achieves on Llama 3 models with FSDP2 on 64 A100 (80GB) GPUs.
+| Techniques | TPS/GPU | Memory(GiB) |
+| ----- | ----: | ----: |
+| FSDP | 5,605 | 67.0 |
+| FSDP + torch.compile | 6,514 | 62.0 |
+| FSDP + torch.compile + Float8 | 8,380 | 61.8 |
 
-| Model size | Batch size | Activation checkpointing | WPS | MFU |
-| ----- | ----- | ----- | ----- | ----- |
-| 8B | 1 | selective layer | 2904 | 56.8% |
-| 8B | 1 | selective op | 2973 | 58.2% |
-| 70B | 1 | full | 331 | 51.7% |
+**Table 4** 2D parallelism (FSDP + TP) + torch.compile + Float8. Llama 3.1 70B model. 256 GPUs (FSDP 32, TP 8). Local batch size 16, global batch size 512. Full activation checkpointing.
 
-We use local batch size 1 (global batch size = local batch size 1 * number of FSDP ranks 64 = 64), because it mimics the small local batch size in large scaled training, and moreoever allows us to compare 1D (FSDP) and 2D (FSDP + TP) training under the same global batch size on both 8B and 70B Llama 3 models, without the out-of-memory (OOM) issue.
+| Techniques | TPS/GPU | Memory(GiB) |
+| ----- | ----: | ----: |
+| 2D | 829 | 71.9 |
+| 2D + AsyncTP | 876 | 67.6 |
 
-Next we show the loss curves for Llama 3 8B and Llama 3 70B training with both 1D parallelism (FSDP2) and 2D parallelism (FSDP2 + Tensor Parallel). All four models are trained the same way as mentioned above with global batch size 64. In terms of activation checkpointing (AC) configs, the Llama 3 8B training jobs use selective op AC, whereas the Llama 3 70B training jobs use full AC. The results are shown in the picture (a TensorBoard screenshot) below.
+**Table 5** 3D parallelism (FSDP + TP + PP) + torch.compile + Float8 + AsyncTP. Llama 3.1 405B model. 512 GPUs (FSDP 8, TP 8, PP8). Local batch size 32, global batch size 256. Full activation checkpointing.
 
-![image](../assets/images/llama3_loss_curves.png)
+| Schedule | TPS/GPU | Memory(GiB) |
+| ----- | ----: | ----: |
+| 1F1B | 100 | 82.5 |
+| Interleaved 1F1B | 128 | 72.7 |
 
-[^1]: We used HBM2e based lower TDP SXM H100(95GB) for our test, the actual peak TFLOPs number is between SXM and NVL, and we don't know its exact value. So this MFU number is lower than actual MFU because we use the peak number of SXM directly.
+**Table 6** 4D parallelism (FSDP + TP + PP + CP) + torch.compile + Float8 + AsyncTP + 1F1B. Llama 3.1 405B model. 512 GPUs (TP 8, PP8). Local batch size 8. Full activation checkpointing.
 
-[^2]: Since for Float8, we are not converting all the matmuls to Float8 because our fused attention implementation is not done in Float8, so this number is lower than expected.
+| Parallelism | Sequence Length | TPS/GPU | Memory(GiB) |
+| ----- | ----: | ----: | ----: |
+| FSDP 8, CP 1 | 32768 | 76 | 75.3 |
+| FSDP 4, CP 2 | 65536 | 47 | 75.9 |
+| FSDP 2, CP 4 | 131072 | 31 | 77.1 |
+| FSDP 1, CP 8 | 262144 | 16 | 84.9 |
+
+
+#### Versions
```

Comment:
minor...maybe "Versions Used For Performance Testing: "
It's overall obvious imo, but making it extra clear what Versions refers to exactly, might ensure no issues.